test/e2e/velero_utils.go

```diff
@@ -195,21 +196,79 @@ func CheckRestorePhase(ctx context.Context, veleroCLI string, restoreName string
 func VeleroBackupNamespace(ctx context.Context, veleroCLI string, backupName string, namespace string) error {
 	backupCmd := exec.CommandContext(ctx, veleroCLI, "create", "backup", backupName, "--include-namespaces", namespace,
 		"--default-volumes-to-restic", "--wait")
+	backupCmd.Stdout = os.Stdout
+	backupCmd.Stderr = os.Stderr
 	fmt.Printf("backup cmd =%v\n", backupCmd)
 	err := backupCmd.Run()
 	if err != nil {
 		return err
 	}
-	return CheckBackupPhase(ctx, veleroCLI, backupName, velerov1api.BackupPhaseCompleted)
+	err = CheckBackupPhase(ctx, veleroCLI, backupName, velerov1api.BackupPhaseCompleted)
+
+	return err
 }
 
 // VeleroRestore uses the veleroCLI to restore from a Velero backup.
 func VeleroRestore(ctx context.Context, veleroCLI string, restoreName string, backupName string) error {
 	restoreCmd := exec.CommandContext(ctx, veleroCLI, "create", "restore", restoreName, "--from-backup", backupName, "--wait")
+	restoreCmd.Stdout = os.Stdout
+	restoreCmd.Stderr = os.Stderr
 	fmt.Printf("restore cmd =%v\n", restoreCmd)
 	err := restoreCmd.Run()
 	if err != nil {
 		return err
 	}
 	return CheckRestorePhase(ctx, veleroCLI, restoreName, velerov1api.RestorePhaseCompleted)
 }
+
+func VeleroInstall(ctx context.Context, veleroNamespace string, cloudProvider string, objectStoreProvider string, useVolumeSnapshots bool,
+	cloudCredentialsFile string, bslBucket string, bslPrefix string, bslConfig string, vslConfig string) error {
+
+	if cloudProvider != "kind" {
```

Comment:
Suggestion: Would it make sense to do this validation upfront before running the tests? e.g adding a flag validation step in `TestE2e`